draft-ietf-tls-dtls13.md

```diff
@@ -2151,6 +2164,14 @@ Client                                             Server
 ~~~
 {: #dtls-key-update title="Example DTLS Key Update"}
 
+Implementations MUST NOT send KeyUpdates that would result
+in epochs of greater than or equal to 2^{48}-1. This limits
```

Comment:
This is somewhat arbitrary.  If the epoch can have 64 bit values, then why not allow for the full range of values to be used?  Is there an analysis that suggests that rekeying too often leads to this problem?

Also, I would have said >2^48-1 OR >= 2^48, but not this.  I can almost guarantee a test failure in some compliance test arising from that choice.  The test will be pointless too.

Reply:
With a key space of 2^{128}, 2^{64} rekeys has a significant chance of key repetition.

The IV randomization is intended to address this, but I prefer to be cautious, especially as there is no realistic scenario in which > 24^{48} rekeys will be needed.

Reply:
Happy to change this to >= 2^{48}

Reply:
If we accept the premise that the nonce is effectively part of the key space, then AES-128 is still probably OK with that many rekeys.  AES-256 and ChaCha20Poly1305 have no such problem.  I'm just concerned that this is arbitrary and unexplained.